### ISSUE TITLE
Timeline for merging full-speed navdata and extra navdata packets into master?

### ISSUE THREAD

**Issue by mikehamer** [AutonomyLab/ardrone_autonomy#51]
Hey @mani-monaj,

Just wondering what the timeline is for merging the dev-unstable changes (eg full-speed navdata and extra navdata packets) into the master branch?

Cheers,
Mike

**Comment by mani-monaj** [comment]
Hi Mike, 

I am back on track with Drones. I will start updating the documentation today. I will do the merge when documentation is updated. 

BTW, I want to freeze development for `ROS Fuerte` after this merge. The plan is to create a branch called `fuerte` from the upcoming master which will be indexed in ROS wiki for Fuerte release, I will then use master branch for Groovy development. I think when next ROS version arrives the similar method can be used to freeze development for Groovy. What do you think?

**Comment by mani-monaj** [comment]
I updated the `dev-unstable` with information about new Navdata, disable auto-hover and Flight animations. I was wondering if you can help me with documentation of the "loop update" section?

**Comment by mani-monaj** [comment]
The documentation and minor code updates on `dev-unstable` has been finished. I tested the branch both on Groovy and Fuerte. I think it is safe now to merge it into the master. Would you please give it a try with your own controllers? @mikehamer @JakobEngel @parcon @sameerparekh

**Comment by sameerparekh** [comment]
I ran a quick test today (I was hoping to run a more extensive test yesterday but something was wrong with my linux configuration and I didn't fix it) and it all works, without using any new features. I also tested the realtime navdata feature, and the video is much less laggy, which I am hoping will lead to more precise control, but I didn't test that really.. I just made sure it flies and tracks.

So +1 on my end.

Thanks.
